db/migrate/20220428214700_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] <b>REQUIRED:</b> And here too, Kindly make sure that the foreign key for the user in the comments table is named `author_id` instead of `user_id`.